src/main/js/components/subcomponents/Order.js

```diff
@@ -2,4 +2,256 @@
 
 /** ----- NPM PACKAGE IMPORTS -----**/
 import React from "react";
-import ReactDOM from "react-dom";
\ No newline at end of file
+import ReactDOM from "react-dom";
+import Select from "react-select";
+import Modal from "react-bootstrap/Modal";
+import Button from "react-bootstrap/Button";
+/**Custom Class imports */
+import { StaffOrders } from "../Staff";
+/** Styling Imports */
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {
+    faTrash, faEdit, faAngleDoubleLeft,
+    faAngleDoubleRight, faAngleLeft, faAngleRight,
+    faPlus
+} from "@fortawesome/free-solid-svg-icons";
+import "../../../resources/static/css/staff.css"
+
+//*******************  IMPORTANT*****
+//Due to an unknown caveat of React.js, the unmounting of this component does not occur for multiple seconds
+//This will therefore cause the end-user to load stale information if the user does not wait long enough in between 
+    //selecting a diningSession's orders to view.
+//It is recommended that the end user, and testers wait 20 seconds after navigating backwards from a diningSessionOrders page before selecting another
+//******/
+//The DiningSessionOrders class is routed to from a list of all diningSession.
+//This component corresponds to a list of Orders associated with a diningSession
+//The hierarchy of Order expression goes as: StaffDiningSession -routes-> DiningSessionOrders -contains-> StaffOrderList -contains-> StaffOrder
+export class DiningSessionOrders extends React.Component{
```

Comment:
Just a suggestion, but for readability should rename this to Orders, or something along that line. A bit confusing to include the name of an entirely different file within this name